lib/Command/CleanupAssistantTasks.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+// SPDX-FileCopyrightText: Sami Finnilä <sami.finnila@nextcloud.com>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+namespace OCA\TpAssistant\Command;
+
+use Exception;
+use OC\Core\Command\Base;
```

Comment:
Is there no OCP equivalent for this?

Reply:
Unfortunately not. Even spreed is using this OC class. https://github.com/nextcloud/spreed/blob/main/lib/Command/Command/ListCommand.php#L26